min_reading_point_count: 2000